src/lib/gossip_net/gossip_net.ml

```diff
@@ -88,7 +91,9 @@ module Make (Message : Message_intf) :
     ; received_reader: Message.content Envelope.Incoming.t Strict_pipe.Reader.t
     ; me: Peer.t
     ; peers: Peer.Hash_set.t
-    ; connections: (Unix.Inet_addr.t, Rpc.Connection.t) Hashtbl.t }
+    ; connections:
```

Comment:
add a comment explaining what this field is for

Reply:
I meant the hashtable with the uuids

Reply:
Oops. Ok